be2-scala/build.sbt

```diff
@@ -1,11 +1,40 @@
+import scala.util.{Try, Success, Failure}
 import sbtsonar.SonarPlugin.autoImport.sonarProperties
+import sbt.IO._
+
 
 name := "pop"
 
 version := "0.1"
 
 scalaVersion := "2.13.5"
 
+
+//Create task to copy the protocol folder to resources
+lazy val copyProtocolTask = taskKey[Unit]("Copy protocol to resources")
+copyProtocolTask := {
+    val log = streams.value.log
+    log.info("Executing Protocol folder copy...")
+    val scalaDest = "be2-scala"
+    baseDirectory.value.name
+    if(! baseDirectory.value.name.equals(scalaDest)){
+        log.error(s"Please make sure you working dir is $scalaDest !")
```

Comment:
very minor point, but is there no way to do away with this ? can't we take the `build.sbt` location as a reference ?
below there are many other files that seem to be referenced with relative paths.

Reply:
Assuming this:
 ..
 |___protocol
 |___be-scala
       |____src/main/resource/protocol
       |____build.sbt
 The project "be2-scala" should be open in IDE such that `build.sbt` is at the root of the workspace and the working directory is `be2-scala` . If not `sbt `does not recognize the build at all.
 
Plus there is no need to worry about this since it's done at compile time. once the jar is properly assembled.